src/coreclr/gc/gcpriv.h

```diff
@@ -4475,6 +4479,9 @@ class gc_heap
     // at the beginning of a BGC and the PM triggered full GCs
     // fall into this case.
     PER_HEAP_ISOLATED_FIELD_DIAG_ONLY uint64_t suspended_start_time;
+    PER_HEAP_ISOLATED_FIELD_DIAG_ONLY uint64_t suspended_end_time;
+    PER_HEAP_ISOLATED_FIELD_DIAG_ONLY uint64_t h0_gc_start_time;
```

Comment:
unless you find these 2 particularly useful I would get rid of them. I added them for very short GCs to see how big a discrepancy there is between the GC time we count for DATAS and the total pause time. and it was clear that for really short GCs they can be quite different. but since we are not acting on this difference, I would just get rid of them